### PR TITLE
Wuerstchen fixes

### PR DESCRIPTION
**Pull request by kashif**
# What does this PR do?

Add some fixes from @dome272  as discussed in slack channel

### CODE REVIEW

File: src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen.py [hunk 1]
```diff
@@ -338,7 +338,7 @@ def __call__(
         # 3. Determine latent shape of latents
         latent_height = int(image_embeddings.size(2) * self.config.latent_dim_scale)
         latent_width = int(image_embeddings.size(3) * self.config.latent_dim_scale)
-        latent_features_shape = (image_embeddings.size(0) * num_images_per_prompt, 4, latent_height, latent_width)
+        latent_features_shape = (image_embeddings.size(0), 4, latent_height, latent_width)
```
Comment (kashif) [inline src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen.py:341]: this is needed to else the decoder pipeline does not work when num_images_per_prompt>1

File: src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen_prior.py [hunk 2]
```diff
@@ -390,6 +398,9 @@ def __call__(
                 generator=generator,
             ).prev_sample
 
+            if callback is not None and i % callback_steps == 0:
+                r = callback(i, t, r)
```
Comment (kashif) [inline src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen_prior.py:402]: this is not right i believe... `r` is not defined

File: docs/source/en/api/pipelines/wuerstchen.md [hunk 3]
```diff
@@ -35,27 +36,18 @@ For the sake of usability Würstchen can be used with a single pipeline. This pi
...
-
-output = pipeline(
-    prompt=caption,
-    height=1024,
+images = pipe(
+    caption, 
     width=1024,
-    negative_prompt=negative_prompt,
+    height=1536,
+    prior_timesteps=DEFAULT_STAGE_C_TIMESTEPS,
```
Comment (sayakpaul) [inline docs/source/en/api/pipelines/wuerstchen.md:48]: Better spit this out here actually. That is what we follow in the official examples.

**Review by sayakpaul — APPROVED** [review]
Looking good. Thanks for the fixes!

**Comment by patrickvonplaten** [comment]
Cool!

**Pull request merged**